src/model/Global.ts

```diff
@@ -578,6 +580,10 @@ echarts.use([${seriesImportName}]);`);
         }
     }
 
+    getCustomRenderer(type: string) {
+        return getCustomSeries(type);
+    }
+
```

Comment:
`ecModel.getCustomRenderer` doesn't seem to be necessary.

`CustomView` can access `customSeriesRegister` and call `getCustomRenderer` directly.